src/cli/restore/exchange.go

```diff
@@ -217,6 +217,7 @@ func restoreExchangeCmd(cmd *cobra.Command, args []string) error {
 	defer utils.CloseRepo(ctx, r)
 
 	dest := control.DefaultRestoreDestination(common.SimpleDateTime)
+	Infof(ctx, "Restoring to:: %s", dest.ContainerName)
```

Comment:
This could be just my OCD, but I'm not sure about `::`. We should probably keep this similar to what we are doing for log file.

Reply:
I too have the same feeling. Added that for testing. Have updated it.